modules/Virtualizer_module/CMakeLists.txt

```diff
@@ -5,9 +5,16 @@
 #
 # set target name
 
-find_package(CybSDK REQUIRED)
 set(EXE_TARGET_NAME VirtualizerModule)
 
+
+find_package(CybSDK REQUIRED)

```

Comment:
Probably you should put ``QUIET`` instead of ``REQUIRED``. When a ``REQUIRED`` package is not found, ``cmake`` should exit with an error.

Reply:
No, I have tested both when found and not found. it is working as it should be. btw, the library is required, so I am not sure if putting quiet is fine.

Reply:
``QUIET`` means that ``cmake`` does not crash and burn if the package is not found and no messages are printed. Nevertheless, you can still have some ``cmake`` machinery later on to complain if you really wanted that dependency.

Reply:
yes, we have also later. but if it is needed for sure this dependency, then it is fine to burn and crash!

Reply:
But then the ``if`` later does not make much sense https://github.com/robotology/walking-teleoperation/blob/70a73b46c5101aaaaedbf3dd9e6c2d5e652a5ea3/modules/Virtualizer_module/CMakeLists.txt#L11-L12 since either is always ``true`` or it should not be reached at all because ``cmake`` should exit with an error. I guess that right now ``REQUIRED`` is not used at all in the ``FindPackage``. I discovered later that you can use ``find_package_handle_standard_args`` to make your custom ``FIndPackage`` behave like a classic one. It is the one throwing errors when the library is not found. You can see an example of usage here https://github.com/iit-danieli-joint-lab/idjl-software/blob/master/cmake/extern/FindMATIO.cmake#L22

Nevertheless, this is not fundamental at the moment, even if it would be cleaner.